### DISCUSSION TITLE
part-ii/2017-59/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#64] [Question comments]
Part II 2017 Fluid Dynamics II: Paper 3, Section II, B 

https://questions.tripos.org/part-ii/2017-59/

**Comment by acornnroca** [comment]
<details><summary>[answers]</summary>

drag force is $$-4 \mu \pi a U$$ but also that's in the notes idk what else to say this entire question is in the notes or is just do the algebra sorry 

</details>
